packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotMessage/UserMessage.tsx

```diff
@@ -19,7 +20,7 @@ Here is an inline code - \`() => void\``;
         timestamp="1 hour ago"
         avatar={customImage}
```

Comment:
This example probably isn't needed, since it was only used to demonstrate how to override the default avatar.